debian/cloud-init-base.postinst

```diff
@@ -111,213 +114,8 @@ for k in sys.argv[2:]:
    fi
 }
 
-handle_preseed_local_cloud_config() {
-   local ccfg=""
-   local cfg_file="/etc/cloud/cloud.cfg.d/90_dpkg_local_cloud_config.cfg"
-   local header="# written by cloud-init-base debian package per preseed entry
-# cloud-init-base/local-cloud-config"
-
-   if process_debconf_setting local-cloud-config; then
-      ccfg="$RET"
-   fi
-
-   if [ "$ccfg" = "_" ]; then
-      rm -f "$cfg_file"
-   elif [ -n "$ccfg" ]; then
-      { echo "$header"; echo "$ccfg"; } > "$cfg_file"
-   fi
-}
-
-fix_1336855() {
-  ### Begin fix for LP: 1336855
-  # fix issue where cloud-init misidentifies the location of grub and
```

Comment:
Cloud-init no longer identifies grub location automatically, and this was to fix an old bug. The debconf values modified here are obsolete.